### ISSUE TITLE
[Firefox] website_suru_DARK jpeg's border is visible since the background color doesn't match

### ISSUE THREAD

**Issue by nobuto-m** [canonical/canonical.com#1201]
## Summary

Please see the following screenshots in Firefox and Chromium respectively. In Firefox, the jpeg's border is distinguishable from the background.

[Firefox]

![image](https://github.com/canonical/canonical.com/assets/4356209/bd14342f-4128-4807-8c66-e83956f1d9ce)

[Chromium]
![image](https://github.com/canonical/canonical.com/assets/4356209/b0f9928a-6ed9-4104-afc3-95f7aab91119)

## Steps to reproduce the behavior

1. Go to https://canonical.com/

## Expected behavior

I suppose no border should be noticeable to users.

## Screenshot

As above.

## Browser/device details

| Name | Firefox |
| --- | --- |
| Version | 123.0.1 |
| Build ID | 20240304153655 |
| Distribution ID | canonical-002 |
| User Agent |   Mozilla/5.0 (X11; Ubuntu; Linux x86\_64; rv:123.0) Gecko/20100101 Firefox/123.0     |
| OS |   `Linux 6.6.0-14-generic #14-Ubuntu SMP PREEMPT\_DYNAMIC Thu Nov 30 10:27:29 UTC 2023`     |
| OS Theme | Yaru / Yaru |
| Application Binary |   /snap/firefox/3941/usr/lib/firefox/firefox     |

**Comment by immortalcodes** [comment]
Hi,
I tried recreating this issue on firefox, but it doesn't seem to happen
![image](https://github.com/user-attachments/assets/3fd7197d-f149-4357-878f-1baad87a0175)

**Comment by nobuto-m** [comment]
Overall there is work to do here. Please don't close it without taking a deeper look.

**Comment by immortalcodes** [comment]
> Overall there is work to do here. Please don't close it without taking a deeper look.

Yes, you are right. I closed it when I wasn't able to reproduce it. I will take a closer look into it.

**Comment by nobuto-m** [comment]
It might be an "issue" specific Firefox when it comes to the interpretation of the color management profile embedded into JPEG. There are a bunch of reports or web pages talking about color management issues specific to Firefox.

In any case, stripping the color profile from the image and sticking to the default sRGB would be the easiest workaround.

**Comment by nobuto-m** [comment]
ref: https://addons.mozilla.org/en-US/firefox/addon/extended-color-management/
> When color pass-through is enabled, Firefox does not make any changes to images or videos before they are passed to the operating system. When color pass-through is disabled, Firefox will use color management to attempt to correct colors in images and video.

And that will flip `gfx.color_management.native_srgb` from false (default) to true. When it's flipped, I don't see the border of the image visible on the web page. So it's definitely a color management thing.

**Comment by lyubomir-popov** [comment]
Given that most people will have firefox set at the default setting without this addon, the only thing I can think of is to ensure the image we display there has an embedded srgb profile (looks like the current one doesn't). Can you try with this one @akbarkz https://assets.ubuntu.com/v1/41749314-test_suru_1.png

**Comment by nobuto-m** [comment]
> Given that most people will have firefox set at the default setting without this addon, the only thing I can think of is to ensure the image we display there has an embedded srgb profile (looks like the current one doesn't). Can you try with this one @akbarkz https://assets.ubuntu.com/v1/41749314-test_suru_1.png

Looks better than the previous one, the border of the image is no longer distinguishable. However, the quality of the gradation is worse than before.

### current
![Screenshot from 2024-08-22 00-32-13](https://github.com/user-attachments/assets/36b6a206-764b-4357-9bb7-28205e8f5295)


### proposed
![Screenshot from 2024-08-22 00-32-01](https://github.com/user-attachments/assets/1ed0e22f-ea92-4341-814f-098d880ab6fc)
